hc/api/management/commands/sendreports.py

```diff
@@ -29,14 +29,15 @@ def add_arguments(self, parser):
 
     def handle_one_run(self):
         now = timezone.now()
-        month_before = now - timedelta(days=30)
+        # Set it to one second to allow those who have been active for more than a second
+        second_before = now - timedelta(seconds=1)
 
         report_due = Q(next_report_date__lt=now)
         report_not_scheduled = Q(next_report_date__isnull=True)
 
         q = Profile.objects.filter(report_due | report_not_scheduled)
         q = q.filter(reports_allowed=True)
-        q = q.filter(user__date_joined__lt=month_before)
+        q = q.filter(user__date_joined__lt=second_before)
```

Comment:
If the minimum time to  receive a report is daily, maybe it would have been better to ensure someone can receive reports as long as you have joined more than a day, but this works too.

Reply:
Seconds was for demo purposes.